AppCenterDistribute/AppCenterDistributeTests/MSDistributeConfigureTests.m

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import "MSChannelGroupProtocol.h"
+#import "MSDistributePrivate.h"
+#import "MSTestFrameworks.h"
+
+static NSString *const kMSTestAppSecret = @"IAMSECRET";
+
+@interface MSDistributeConfigureTests : XCTestCase
+
+@end
+
+@implementation MSDistributeConfigureTests
+
+- (void)testConfigureFlagsBeforeStart {
+
+  // If
+  [MSDistribute sharedInstance].distributeFlags = -1;
```

Comment:
Why setting `distributeFlags` to an invalid value like `-1`? 

`XCTAssertEqual(MSDistributeFlagsDisableAutomaticCheckForUpdate, [MSDistribute sharedInstance].distributeFlags);`

I don't understand how this test would work, `1 ^ -1` should result in `-2` not `1`.

Reply:
This test is to make sure the flag got changed by the API call. `-1` won't be used but it just want to make sure of the value change only in unit test.

Reply:
> I don't understand how this test would work, 1 ^ -1 should result in -2 not 1.
There is no this kind of operations during the test.

Reply:
You are right. Sorry for the confusion, I misunderstood the code.